src/test/java/com/itkonboarding/airport_gate/controllers/FlightControllerTest.java

```diff
@@ -0,0 +1,118 @@
+package com.itkonboarding.airport_gate.controllers;
+
+import com.itkonboarding.airport_gate.dto.request.FlightRequestDto;
+import com.itkonboarding.airport_gate.dto.request.FlightUpdateRequestDto;
+import com.itkonboarding.airport_gate.dto.response.FlightResponseDto;
+import com.itkonboarding.airport_gate.entities.Flight;
+import com.itkonboarding.airport_gate.exceptions.ResourceNotFoundException;
+import com.itkonboarding.airport_gate.mappers.FlightMapper;
+import com.itkonboarding.airport_gate.services.FlightService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Random;
+
+import static com.itkonboarding.airport_gate.exceptions.ErrorCode.FLIGHT_NOT_FOUND;
+import static java.util.Optional.empty;
+import static java.util.Optional.of;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class FlightControllerTest {
+
+    @Mock
+    private FlightService flightService;
+
+    @Mock
+    private FlightMapper flightMapper;
+
+    @InjectMocks
+    private FlightController flightController;
+
+    @Test
+    void get_notFoundExtension() {
+        var id = new Random().nextInt();
+        var flight = mock(Flight.class);
+
+        when(flightService.findById(id)).thenReturn(empty());
+
+        assertAll(
+                () -> assertThatExceptionOfType(ResourceNotFoundException.class)
+                        .isThrownBy(() -> flightController.get(id))
+                        .withMessage(FLIGHT_NOT_FOUND),
+                () -> verify(flightMapper, never()).flightToFlightResponseDto(flight)
```

Comment:
It is more elegant solution to use veirfyNoInteraction(flightMapper) here. U don't have to fix this.